src/internal/transactionenv/env.go

```diff
@@ -281,21 +233,11 @@ func (t *directTransaction) UpdateJobState(original *pps.UpdateJobStateRequest)
 	return t.txnCtx.txnEnv.ppsServer.UpdateJobStateInTransaction(t.txnCtx, req)
 }
 
-func (t *directTransaction) ModifyRoleBinding(original *auth.ModifyRoleBindingRequest) (*auth.ModifyRoleBindingResponse, error) {
```

Comment:
These weren't commonly used and you can just call the InTransaction methods directly.